GsaGH/Helpers/PostHog.cs

```diff
@@ -10,6 +10,47 @@ internal class PostHog {
       _ = OasysGH.Helpers.PostHog.SendToPostHog(PluginInfo.Instance, eventName, properties);
     }
 
+    internal static void Diagram(
+      string diagramType, CaseType caseType, string type, GsaAPI.DiagramType subType, EntityType entityType) {
```

Comment:
this should call the other method

Reply:
fixed with: https://github.com/arup-group/GSA-Grasshopper/pull/537/commits/abe8bd85c9a811e487834e5657d2efdea19f4a4a